job-service/src/main/resources/jobtype/taskscript-schema.yaml

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
This is in the `jobtype` package.  Use the same package name as the related source code - i.e. `com.hpe.caf.services.job.jobtype`.